core/domain/feedback_services_test.py

```diff
@@ -89,124 +90,10 @@ def test_status_of_newly_created_thread_is_open(self):
             self.assertEqual(thread_status, feedback_models.STATUS_CHOICES_OPEN)
 
     def test_get_exp_id_from_thread_id(self):
-        thread_id = 'exp1.1234'
-        self.assertEqual(
-            feedback_services.get_exp_id_from_thread_id(thread_id), 'exp1')
-
-
-class SuggestionQueriesUnitTests(test_utils.GenericTestBase):
```

Comment:
Where are the corresponding tests for the "True" case (to ensure that things still work fine after the flag is flipped?

Reply:
Same as above (regarding old framework)

Reply:
See my response above too.

Reply:
Added a comment